thinc/layers/embed.py

```diff
@@ -53,14 +53,16 @@ def backprop(d_output: OutT) -> InT:
     return output, backprop
 
 
-def create_init(initializer: Callable) -> Callable:
-    def init(
-        model: Model[InT, OutT], X: Optional[InT] = None, Y: Optional[OutT] = None
+class CreateInit(object):
+    """Create an init function, given a dictionary of parameter initializers."""
+
+    def __init__(self, initializer: Callable):
+        self.initializer = initializer
+
+    def init(self, model: Model[InT, OutT], X: Optional[InT] = None, Y: Optional[OutT] = None
```

Comment:
```suggestion
    def __call__(self, model: Model[InT, OutT], X: Optional[InT] = None, Y: Optional[OutT] = None
```

We should just make it callable, to avoid the `.init`

Reply:
Yes, ofcourse, that makes much more sense!